package.json

```diff
@@ -63,9 +63,6 @@
   },
   "lint-staged": {
     "src/**/*.js": [
-      "prettier-eslint --write --log-level error",
-      "eslint --fix",
```

Comment:
Shouldn't it still run `eslint` just not `--fix`?

Reply:
You're right